slurp.1.scd

```diff
@@ -68,7 +68,7 @@ Interpreted sequences are:
 
 %l	Label included with region from stdin
 
-The default format is "%x,%y %wx%h".
+The default format is "%x,%y %wx%h\n".
```

Comment:
This renders as `%x,%y %wx%hn`. We need to escape the `\`.

Reply:
resolved in 9966d67